composer.json

```diff
@@ -48,7 +48,8 @@
     "symfony/proxy-manager-bridge": "~2.4",
     "phpunit/phpunit": "^4.8",
     "matthiasnoback/symfony-config-test": "^2.0",
-    "matthiasnoback/symfony-dependency-injection-test": "^1.1"
+    "matthiasnoback/symfony-dependency-injection-test": "^1.1",
+    "broadway/broadway-saga": "^0.2"
```

Comment:
Without it, I get 
```
1) Broadway\Bundle\BroadwayBundle\DependencyInjection\RegisterSagaStateRepositoryCompilerPassTest::it_sets_the_saga_state_repository_alias
ReflectionException: Class Broadway\Saga\State\RepositoryInterface does not exist
```
since the interface is defined in the `broadway-saga` package.

I think we could define a dummy interface in another file and require this for the test but it feels dirty.

Do you have a another, better idea to make the interface available?